packages/integrations/test/pi-hole.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { GenericContainer } from "testcontainers";
+import { describe, expect, test } from "vitest";
+
+import { PiHoleIntegration } from "../src";
+
+describe("Pi-hole integration", () => {
+  test("getSummaryAsync should return summary from pi-hole", async () => {
+    // Arrange
+    const password = "12341234";
+    const apiKey = "3b1434980677dcf53fa8c4a611db3b1f0f88478790097515c0abb539102778b9";
+
+    const piholeContainer = await new GenericContainer("pihole/pihole:latest")
+      .withEnvironment({
+        WEBPASSWORD: password,
+      })
+      .withExposedPorts(80)
+      .start();
+
+    // We need to wait until pi hole has actually started and is ready to get requests
+    await sleep(5000);
+
+    const piHoleIntegration = new PiHoleIntegration({
+      id: "1",
+      decryptedSecrets: [
+        {
+          kind: "apiKey",
+          value: apiKey,
+        },
+      ],
+      name: "Pi hole",
+      url: `http://${piholeContainer.getHost()}:${piholeContainer.getMappedPort(80)}`,
+    });
+
+    // Act
+    const result = await piHoleIntegration.getSummaryAsync();
+
+    // Assert
+    expect(result.adsBlockedToday).toBe(0);
```

Comment:
I would expect a test with a value other than 0 because 0 is a known default value in a lot of places

Reply:
Yes that's true, this also why I made the domains blocked greater than one. The issue is that when you spin up a new instance of pi hole there is no blocked ad yet. I would suggest to leave it as it is with the 3 other values added now